### ISSUE TITLE
Error in localization for Traditional Chinese (zh-TW) breaks settings dialog.

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#130]
In the zh-TW version of SmartTemplates, the settings screen cannot be displayed. Instead if shows the following error:

```
XML 解析錯誤: 處理外部 Entity 參考時發生錯誤
位置: chrome://smarttemplate4/content/settings.xhtml
行號 4, 第 94 個字:
   <!ENTITY % localeDTD SYSTEM "chrome://smarttemplate4/locale/smartTemplate-settings.dtd">  %localeDTD;
---------------------------------------------------------------------------------------------^
```

**Comment by RealRaven2000** [comment]
Here is a hotfix for Thunderbird 78:

[smartTemplate-fx-3.5pre23.zip](https://github.com/RealRaven2000/SmartTemplate4/files/6364955/smartTemplate-fx-3.5pre23.zip)

To install this version, download the zip file. Then drag the zip file into Thunderbird Add-ons Manager.

**Comment by RealRaven2000** [comment]
Fixed in 3.5.1 released on 21/05/2021
